docs/production/upgrade-or-modify.md

```diff
@@ -321,6 +383,10 @@ instructions for other supported platforms.
    /home/zulip/deployments/current/scripts/setup/reindex-textual-data --force
    ```
 
+7. [Upgrade from Ubuntu 20.04 to
+   22.04](#upgrading-from-ubuntu-2004-focal-to-2204-jammy), so that
+   you are running a supported operating system.
```

Comment:
Ubuntu 20.04 is still a supported operating system, so this should be rephrased to something like “We recommend that you now [upgrade from Ubuntu 20.04 to 22.04]. Although Ubuntu 20.04 is still supported at this time, Ubuntu 22.04 will be supported farther into the future.”

Reply:
Excellent, thank you!